controller/user/manager.py

```diff
@@ -92,7 +92,7 @@ def remove_organization_from_user(user_mail: str) -> None:
 
 def get_active_users(minutes: str, order_by_interaction: bool) -> User:
     now = datetime.now()
-    last_interaction_range = (now - timedelta(minutes=minutes)) if minutes else None
+    last_interaction_range = (now - timedelta(minutes=minutes)) if minutes > 0 else None
```

Comment:
can not compare minutes:str > 0, at runtime it also gets a int, so ther eis no error, but maybe change types